server/views/case-summary-record-order.njk

```diff
@@ -218,76 +221,85 @@
-            <div class="govuk-grid-column-one-quarter">
-                <h3 class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0">Acceptable</h3>
+            <div class="govuk-grid-column-one-third pac-!-grid-column-18">
+                <h3 class="govuk-heading-m govuk-!-margin-bottom-0"><span
+                            class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">
+                Acceptable</span></h3>
             </div>
-            <div class="govuk-grid-column-three-quarters">
-                <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-0"><span
-                            class="govuk-heading-m govuk-!-margin-right-2 govuk-!-display-inline-block govuk-!-margin-right-2 govuk-!-margin-bottom-1 app-dashboard-count">{{ attendedAppointments.complied.flat(Infinity) | length }}</span>Attendances
+            <div class="govuk-grid-column-two-thirds">
+                <p class="govuk-heading-m govuk-!-margin-bottom-1"><span
+                            class="pac-dashboard-count">{{ attendedAppointments.complied.flat(Infinity) | length }}</span><span
+                            class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">Attendances</span>
                 </p>
-
                 {% for complied in attendedComplied | sort(true, false, 'count') %}
                     <p class="govuk-body govuk-!-margin-0"><span
-                                class="govuk-body govuk-!-margin-right-2 app-dashboard-count">{{ complied.count }}</span> {{ complied.description }}
+                                class="govuk-body pac-dashboard-count">{{ complied.count }}</span>{{ complied.description }}
                     </p>
                 {% endfor %}
-
-                <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-0 govuk-!-margin-top-2"><span
-                            class="govuk-heading-m govuk-!-margin-right-2 govuk-!-display-inline-block govuk-!-margin-right-2 govuk-!-margin-bottom-1 app-dashboard-count">{{ absentAppointments.complied.flat(Infinity) | length }}</span>Absences
-                </p>
-
+                <p class="govuk-heading-m govuk-!-margin-bottom-1"><span
+                            class="pac-dashboard-count">{{ absentAppointments.complied.flat(Infinity) | length }}</span><span
+                            class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">
+                Absences </span></p>
                 {% for complied in absentComplied | sort(true, false, 'count') %}
                     <p class="govuk-body govuk-!-margin-0"><span
-                                class="govuk-body govuk-!-margin-right-2 app-dashboard-count">{{ complied.count }}</span> {{ complied.description }}
+                                class="govuk-body- pac-dashboard-count">{{ complied.count }}</span>{{ complied.description }}
```

Comment:
There's an additional - after govuk-body

Reply:
Yep I'll sort it, good spot